tests/terraform/checks/resource/oci/example_SecurityListIngressStateless/main.tf

```diff
@@ -0,0 +1,28 @@
+resource "oci_core_security_list" "pass" {
+    compartment_id = oci_identity_compartment.tf-compartment.id
+    vcn_id = oci_core_vcn.test_vcn.id
+    ingress_security_rules {
+        protocol = "all"
+        source="192.168.1.0/24"
+    }
+}
+
+resource "oci_core_security_list" "pass2" {
+    compartment_id = oci_identity_compartment.tf-compartment.id
+    vcn_id = oci_core_vcn.test_vcn.id
+    ingress_security_rules {
+        protocol = "all"
+        source="192.168.1.0/24"
+        stateless = true
+    }
+}
+
+resource "oci_core_security_list" "fail" {
+  compartment_id = oci_identity_compartment.tf-compartment.id
+  vcn_id         = oci_core_vcn.test_vcn.id
+      ingress_security_rules {
+        protocol = "all"
+        source="192.168.1.0/24"
+        stateless = false
+    }
```

Comment:
@JamesWoolfenden Can you please add a resource where `ingress_security_rules` doesn't exist?

Reply:
added